test/e2e/fixture/fixture.go

```diff
@@ -904,9 +904,9 @@ func RestartRepoServer() {
 		}
 		FailOnErr(Run("", "kubectl", "rollout", "restart", "deployment", workload))
 		FailOnErr(Run("", "kubectl", "rollout", "status", "deployment", workload))
```

Comment:
```suggestion
		FailOnErr(Run("", "kubectl", "rollout", "status", "deployment", workload))
		// wait longer to avoid error on s390x
		time.Sleep(10 * time.Second)
```